common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -906,47 +905,79 @@ div.problem {
 // +Problem - Actions
 // ====================
 div.problem .action {
-  margin-top: $baseline;
+  @include margin($baseline 0 $baseline 0);
+  min-height: $baseline;
 
-  .save, .check, .show, .reset, .hint-button {
-    @include margin-right($baseline/2);
-    margin-bottom: ($baseline/2);
-    height: ($baseline*2);
-    vertical-align: middle;
-    text-transform: uppercase;
-    font-weight: 600;
-    @media print {
-      display: none;
+  // override default button hover
+  button {
+    &:hover {
+      background-image: none;
+      box-shadow: none;
+    }
+    &:focus {
+      box-shadow: none;
+    }
+    &.btn-default {
+      background-color: transparent;
+    }
+    &.btn-brand {
+      &:hover {
+        background-color: #0ea6ec;
```

Comment:
Are these color values (here and below) from the pattern library? If so, it would be better to define variables that reference the pattern library, similar to https://github.com/edx/edx-platform/pull/12950/files#diff-8e9589dd2238bbdf2a4fca2969a3e963R44.